docs/source/daml/intro/4_Transformations.rst

```diff
@@ -76,13 +76,13 @@ Choices in the Ledger Model
 
 In :doc:`1_Token` you learned about the high-level structure of a DAML ledger. With choices and the `exercise` function, you have the next important ingredient to understand the structure of the ledger and transactions.
 
-A *transaction* is a list of *actions*, and there are just three kinds of action: ``create``, ``exercise`` and ``fetch``. All actions are performed on a contract.
+A *transaction* is a list of *actions*, and the most common kinds of action are ``create``, ``exercise`` and ``fetch``.
```

Comment:
https://docs.daml.com/concepts/ledger-model/ledger-structure.html also mentions _key assertions_ . `create` is not "perfomed on a contract" imho.

Reply:
I'd list all action types then, since there's only one more (otherwise, it's unclear if there are 50 other ones). The crappy part is that the ledger model and the implementation diverge here, as the implementation has a `LookupByKey` action, where's the ledger model only has `Fetch` and `NoSuchKey`. This should however be resolved when https://github.com/digital-asset/daml/pull/4029 lands

Reply:
ok, I added ``key assertion``, just copied it from Ledger model docs.